src/modules/search-result/search-result.jsx

```diff
@@ -103,7 +103,9 @@ const SearchResult = ({
         )}
       </figure>
 
-      <div className={styles.content}>{children}</div>
+      <div className={styles.content} itemProp="abstract">
```

Comment:
It will not always be abstract that's why it's not added. It can be either a text snipper or an abstract otherwise depending on the context.